<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostureSelectionDialog</class>
 <widget class="QDialog" name="PostureSelectionDialog">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>0</y>
    <width>52</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_open">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/light/posture_open</normaloff>:/light/posture_open</iconset>
     </property>
     <property name="themeIconName" stdset="0">
      <string notr="true">posture_open</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Open</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="ToolWindowButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_closed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/light/posture_closed</normaloff>:/light/posture_closed</iconset>
     </property>
     <property name="themeIconName" stdset="0">
      <string notr="true">posture_closed</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Closed</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="ToolWindowButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_tent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/light/posture_tent</normaloff>:/light/posture_tent</iconset>
     </property>
     <property name="themeIconName" stdset="0">
      <string notr="true">posture_tent</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Tent</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="ToolWindowButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_flipped">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/light/posture_flipped</normaloff>:/light/posture_flipped</iconset>
     </property>
     <property name="themeIconName" stdset="0">
      <string notr="true">posture_flipped</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Flipped</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="ToolWindowButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_halfopen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/light/posture_half-open</normaloff>:/light/posture_half-open</iconset>
     </property>
     <property name="themeIconName" stdset="0">
      <string notr="true">posture_half-open</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Half-Open</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="ToolWindowButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
